app/javascript/src/components/Authentication/Signup.jsx

```diff
@@ -6,8 +6,9 @@ import { Input } from "neetoui/formik";
 import PropTypes from "prop-types";
 
 import authenticationApi from "apis/authentication";
-import formInitialValues from "constants/formInitialValues";
-import formValidationSchemas from "constants/formValidationSchemas";
+
+import formInitialValues from "./constants/formInitialValues";
+import formValidationSchemas from "./constants/formValidationSchemas";
```

Comment:
Keep these are constants within a single `constants.js` file. Currently there is not enough content so as to split them out into separate files.